docs/monitoring/slack.md

```diff
@@ -0,0 +1,21 @@
+# Slack for notifications
+
+commcare-cloud can be configured to send event notifications to Slack such as
+pre and post deploy messages.
+
+To configure commcare-cloud to send messages to Slack:
+
+1. Create a [Slack app](https://api.slack.com/authentication/basics) and copy the
+access token.
+2. Set the access token as a commcare-cloud secret:
+
+    Run the following command and paste in the token when prompted:
+    ```shell
+    commcare-cloud <env> secrets edit slack_token
```

Comment:
My intention when writing the `secrets_backend` changes was to keep a list of all possible secrets in https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/environment/secrets/secrets.yml. I forget if I made `commcare-cloud <env> secrets edit slack_token` actually fail if it's not available, but apparently I didn't?

Reply:
https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/README.md#secrets-management is the closest thing I wrote to docs about this, I think.

Reply:
acb9cb1ad